dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -140,13 +142,14 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     @SuppressWarnings("unchecked")
     public <T extends WithId> ListResult<T> fetchAll(Class<T> model, Function<ListResult<T>, ListResult<T>>... operators) {
-        Kind kind = Kind.from(model);
```

Comment:
Its not unrelated.  The caching changes have to go otherwise the tests would be breaking.